src/class-wpml-gutenberg-integration.php

```diff
@@ -76,25 +76,27 @@ function register_strings( WP_Post $post, $package_data ) {
 	private function register_blocks( $blocks, $package_data ) {
```

Comment:
@BruceOnTheGo can you add `array` type-hint to both arguments, please?

Reply:
In the PHPDoc, it would help to be a bit more specific and define `@param WP_Block_Parser_Block[] $blocks`